dhall/src/Dhall/Pretty/Internal.hs

```diff
@@ -159,18 +160,22 @@ renderSrc
     -> Doc Ann
 renderSrc strip (Just (Src {..}))
     | not (Text.all isWhitespace srcText) =
-        Pretty.align (Pretty.concatWith f newLines <> suffix)
+        renderComment (strip srcText)
+renderSrc _ _ =
+    mempty
+
+renderComment :: Text -> Doc Ann
```

Comment:
Could you add a haddock here? More for us (maintainers) than for hackage. I see that it assumes that the `Text` should be stripped by checking the implementation, but could be great to just read it from its haddock.